ConsecutiveNumber.java

```diff
@@ -0,0 +1,43 @@
+/***
+ * Suman Kumari
+ * 09-09-2024
+ * Java program that finds whether a given number can be expressed as the sum of two or
+more consecutive natural numbers. If possible, the program should print all possible
+combinations of consecutive natural numbers that sum up to the given number.
+ */
+
+package Week_2_Assignment_1;
+
+import java.util.Scanner;
+
+public class ConsecutiveNumber {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		while(true) {
+			System.out.print(Constant.ENTER_NUMBER);
+			int n = sc.nextInt();
+			if(n <= 0) {
+	        	System.out.println("Invalid Number");
+	        }
+			else {
+				System.out.println(Constant.CONSECUTIVE_NUMBER + n + ":");
+				ConsecutiveSum(n);
+			}
+		}
+	}
+	public static void ConsecutiveSum(int n) {

```

Comment:
describe each method in brief through comments including short definition , input parameters and return values .